hello-spring/descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id": "@artifactId@-@version@",
+    "name": "Hello World",
+    "provides": [
+        {
+            "id": "hello",
+            "version": "1.0",
+            "handlers": [
+                {
+                    "methods": ["GET", "POST"],
+                    "pathPattern": "/hello",
+                    "permissionsRequired": []
+                }
+            ]
+        },
+        {
+            "id": "_tenant",
+            "version": "2.0",
+            "interfaceType": "system",
+            "handlers": [
+                {
+                    "methods": [ "POST" ],
+                    "pathPattern": "/_/tenant"
+                },
+                {
+                    "methods": [ "GET", "DELETE" ],
+                    "pathPattern": "/_/tenant/{id}"
+                }
+            ]
+        }
+    ],
+    "launchDescriptor": {
+        "dockerImage": "@artifactId@",
+        "dockerPull": false,
+        "dockerArgs": {
+            "HostConfig": {
+                "PortBindings": {
+                    "8080/tcp": [
+                        {
+                            "HostPort": "%p"
+                        }
+                    ]
+                }
+            }
+        },
+        "env": [
+            { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0" },
+            { "name": "DB_HOST", "value": "postgres" },
+            { "name": "DB_PORT", "value": "5432" },
+            { "name": "DB_USERNAME", "value": "folio_admin" },
+            { "name": "DB_PASSWORD", "value": "folio_admin" },
+            { "name": "DB_DATABASE", "value": "okapi_modules" },
+            { "name": "DB_QUERYTIMEOUT", "value": "60000" },
+            { "name": "DB_CHARSET", "value": "UTF-8" },
+            { "name": "DB_MAXPOOLSIZE", "value": "5" }
+        ]
+    }
+}
```

Comment:
Please keep the indentation [style](https://dev.folio.org/guidelines/contributing/#coding-style) like the other modules.